src/applications/_mock-form-ae-design-patterns/mocks/server.js

```diff
@@ -0,0 +1,245 @@
+const delay = require('mocker-api/lib/delay');
+
+const user = require('./endpoints/user');
+const handleUserUpdate = require('./endpoints/user/handleUserUpdate');
+
+const address = require('./endpoints/address');
+const emailAddress = require('./endpoints/email-addresses');
+const phoneNumber = require('./endpoints/phone-number');
+
+const { generateFeatureToggles } = require('./endpoints/feature-toggles');
+
+const serviceHistory = require('./endpoints/service-history');
+
+const maintenanceWindows = require('./endpoints/maintenance-windows');
+
+const {
+  prefill,
+} = require('./endpoints/in-progress-forms/mock-form-ae-design-patterns');
+
+// transaction status that is used for address, email, phone number update flows
+const {
+  getEmptyStatus,
+  generateStatusResponse,
+} = require('./endpoints/status');
+
+// use one of these to provide a generic error response for any endpoint
+const genericErrors = {
+  error500: require('./errors/500.json'),
+  error401: require('./errors/401.json'),
+  error403: require('./errors/403.json'),
+};
+
+// seed data for VAMC drupal source of truth json file
+const mockLocalDSOT = require('./script/drupal-vamc-data/mockLocalDSOT');
+
+// utils
+const {
+  delaySingleResponse,
+  logRequest,
+  requestHistory,
+  boot,
+} = require('./script/utils');
+
+const responses = {
+  'GET /v0/in_progress_forms/FORM_MOCK_AE_DESIGN_PATTERNS': (_req, res) => {
+    return res.json({
+      formData: {
+        data: {
+          attributes: {
+            veteran: {
+              address: {
+                addressLine1: '623 Lesser Dr',
+                city: 'Fort Collins',
+                stateCode: 'CO',
+                zipCode5: '80524',
+                countryName: 'USA',
+              },
+              firstName: 'John',
+              lastName: 'Donut',
+              middleName: 'Jelly',
+              phone: {
+                areaCode: '970',
+                phoneNumber: '5561289',
+              },
+              emailAddressText: 'sample@email.com',
+            },
+          },
+        },
+        nonPrefill: {
+          veteranSsnLastFour: '3607',
+          veteranVaFileNumberLastFour: '3607',
+        },
+      },
+      metadata: {
+        version: 0,
+        prefill: true,
+        returnUrl: '/veteran-details',
+      },
+    });
+  },
```

Comment:
@adamwhitlock1 Added this GET request here and deleted it from the profile mock server.